pages/api/graphql/index.ts

```diff
@@ -1,36 +1,37 @@
 import { withCors } from "middleware";
 import { NextApiRequest, NextApiResponse } from "next";
-import { getSession } from "next-auth/react";
+import { getToken } from "next-auth/jwt";
 import getConfig from "next/config";
 import { Octokit } from "octokit";
 
-import { CustomSession } from "interfaces/custom-session";
+import { error as LogError } from "helpers/api/handle-log";
 
-const { serverRuntimeConfig: { github: { token: botToken } } } = getConfig();
+const { serverRuntimeConfig: { authSecret, github: { token: botToken } } } = getConfig();
 
 async function post(req: NextApiRequest, res: NextApiResponse) {
-  try {
-    const { query, params } = req.body;
-
-    const session = await getSession({ req }) as CustomSession;
+  const { query, params } = req.body;
+  const { access_token, login, name } = await getToken({ req, secret: authSecret });
 
+  try {
     const octokit = new Octokit({
-      auth: session?.user?.accessToken || botToken
+      auth: access_token || botToken
     });
 
     const result = await octokit.graphql(query, params);
 
     return res.status(200).json(result);
   } catch(error) {
-    error("", { req, error });
-    return res.status(500).json(error);
+    LogError("", { req, error, access_token, login, name });
```

Comment:
add `GraphQL Proxy` as a message so we can _eventually_ find it on the elastic search